clusterloader2/testing/network-policy/network-policy-soak/Readme.txt

```diff
@@ -0,0 +1,18 @@
+# test setup details - 
```

Comment:
Rename file to README.md. Add documentation for different type of network policies, like what is k8s, ccnp, cnp.